webhook/webhook.go

```diff
@@ -177,34 +168,8 @@ func (ac *Webhook) Run(stop <-chan struct{}) error {
 	}
 
 	logger.Info("Found certificates for webhook...")
-	if ac.Options.RegistrationDelay != 0 {
```

Comment:
what happened to all the registration delay code?

Reply:
it'd gone 💥 

A delay makes some sense when it's a one-time thing, but it's effectively continuous now.